tracer/build/_build/Build.Steps.cs

```diff
@@ -1193,10 +1193,19 @@ void RunWindowsIisIntegrationTests(Project project)
                 .Concat(securitySampleProjects)
                 .Concat(regressionProjects)
                 .Concat(instrumentationProjects)
-                .Where(path =>
+                .Select(path => (path, project: Solution.GetProject(path)))
```

Comment:
This part starts to be slightly complicated. I assume we could simplify it and just build all samples if we could build samples in // of the build. Maybe we should prioritize this work.

Reply:
> This part starts to be slightly complicated

"Starts" 😂 

>  I assume we could simplify it and just build all samples if we could build samples in // of the build. 

Potentially. That's a lot harder than you would like though, and will likely slow the build (we can't currently build some of the samples until after we've built the tracer). And that won't _reduce_ the complexity in a lot of cases, it'll just shift it around. Maybe, it's worth it, but I don't feel like it should be a priority